uwblueprint/focus-on-nature
# Create calendar component in camp creation scheduled sessions

## Notion ticket link

[Camp Creation/Editing: Scheduling Sessions Calendar Component](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=97be9a874e95473786f511f136ac6dd6&pm=c)



## Implementation description
*  **Used https://fullcalendar.io/** (all devs will have to rebuild on merge)
*  Custom header component to have our styling for today button, and Chakra input selectors
* Moved calendar / side panel components into separate directories



## Steps to test
1. Create camp and add sessions with different dates, does this match what we expect? Test case with 8+ sessions to make sure colours loop properly
2. Check navigation functionality by going between months and years, select today


https://user-images.githubusercontent.com/40974372/202888747-e3d6ad6b-0178-443c-acfd-55ca319ac1b9.mov



## What should reviewers focus on?
* Some messy CSS handling + responsiveness that people may want to take a look at (should check out the 1980x1080px dimensions)


## Checklist
